cpp/src/svm/svm_api.cpp

```diff
@@ -196,7 +196,7 @@ cumlError_t cumlDpSvcPredict(cumlHandle_t handle, double *input, int n_rows,
   if (status == CUML_SUCCESS) {
     try {
       ML::SVM::svcPredict(*handle_ptr, input, n_rows, n_cols, kernel_param,
-                          model, preds, buffer_size);
+                          model, preds, buffer_size, true);
```

Comment:
Question: should we also expose a `svcDecisionFunction` function on our C-API?

Reply:
We should give the possibility to get the decision function value also in the C-API, you are right. The question is whether to do it with a separate function, or just expose the last arg of `svcPredict` to `cumlSpSvcPredict`. The latter would be less code, I would go for that. What is your opinion?

Reply:
Agree with you. For now. exposing the last arg would be sufficient.